saleor/graphql/order/filters.py

```diff
@@ -27,7 +27,7 @@
 
 def filter_payment_status(qs, _, value):
     if value:
-        qs = qs.filter(payments__is_active=True, payments__charge_status__in=value)
+        qs = qs.filter(payments__charge_status__in=value)
```

Comment:
@korycins what do you think? Won't we end up with plenty of payments that are irrelevant? Maybe instead of removing `payments__is_active=True` condition, we should handle this specific case?

Reply:
Also, it could be rewritten using `Exists` and `OuterRef`

Reply:
If you will filter out by `NOT_CHARGED` you will probably get incorrect orders (for example, order with two payments, one active&captured, second disabled&NOT_CHARGED (the default status). 
I would suggest to extract  `FULLY_REFUNDED` to separate if condition.